### ISSUE TITLE
LOC is way off, possibly due to mass file deletion?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waterxi/svnplot#34]
```
My repository originally contained my project, and a 'lib' folder, which
contained various external libraries. 

At some point I removed the lib folder from the repository.

SVN Plot appears to be still counting lines of code in this folder, or for
some other reason it's count is way off.

SVNPlot shows 900,000+ lines of code, and more tellingly shows no drop in
the number of lines when I know that several hundred lines should have been
removed when I remover the 'lib' folder.

running cloc (a line of code counting app) on a freshly checked out copy of
my repository shows around 90,000 lines... so SVN Plot appears to be an
order of magnitude out!!


```

Original issue reported on code.google.com by `deic...@googlemail.com` on 9 Mar 2010 at 4:04

**Comment by GoogleCodeExporter** [comment]
```
Directory renames/deletes are not correctly accounted in SVNPlot 0.5.x. I am 
working on the fix. It will be available in 0.6.
```

Original comment by `nitinbh...@gmail.com` on 9 Mar 2010 at 5:32
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
initial fix is available in trunk now.
```

Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 4:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 12 Mar 2010 at 5:02

**Comment by GoogleCodeExporter** [comment]
```
fixed in 0.6 release.
```

Original comment by `nitinbh...@gmail.com` on 5 Apr 2010 at 3:27
- Changed state: **Fixed**
